### PR TITLE
NTP-443: Added configurable GTM integration

### PR DESCRIPTION
**Pull request by iterative-it**
## Context

The team would like to evaluate the extra control Google Tag Manager provides for tracking additional events. This spike evaluates the GTM onboarding process and code changes

## Link to Jira ticket

https://dfedigital.atlassian.net/browse/NTP-443

## Things to check

- [X] Title is in the format "NTP-XXX: Short description" where NTP-XXX is the Jira ticket reference
- [X] Code and tests follow the [coding standards](/docs/coding-standards.md)
- [X] `dotnet format` has been run in the repository root
- [ ] Test coverage of new code is at least 80%
- [ ] All UI related acceptance criteria specified in the ticket have been added to the relevant [cypress test feature file(s)](/UI/cypress/e2e/)
- [X] Unless explicitly mentioned in the ticket, all UI work should have been tested as working on desktop and mobile resolutions with JavaScript on and off
- [ ] Database migrations can be applied to the current codebase in main without causing exceptions
- [ ] Debug logging has been added after all logic decision points
- [X] All [automated testing](/README.md#testing) including accessibility and security has been run against your code
- [ ] **PR deployment has been signed off by wider team**

### CODE REVIEW

**Review by stephenjmcneill — APPROVED** [review]

File: UI/Pages/TuitionPartner.cshtml [hunk 1]
```diff
@@ -205,7 +205,7 @@
                         Website
                     </govuk-summary-list-row-key>
                     <govuk-summary-list-row-value>
-                        <a href="@Model.Data.Website" class="govuk-link" target="_blank" data-testid="tuition-partner-website-link">@Model.Data.Website</a>
+                        <a id="app-tuition-partner-website-link" href="@Model.Data.Website" class="govuk-link" target="_blank" data-testid="tuition-partner-website-link">@Model.Data.Website</a>
```
Comment (benagain) [inline UI/Pages/TuitionPartner.cshtml:208]: Are these link IDs relevant to the GTM integration?

**Review by benagain — APPROVED** [review]
Looks fine as far as I can tell!
